### PR TITLE
[DOCS-435] SAML SSO: Okta as IdP

### PR DESCRIPTION
**Pull request by ana-dashuk-cobalt**
### Added

| Page | Deploy Preview | Comment |
| ----- | ----- | ----- |
| How to Configure SAML 2.0 for Cobalt | [Link](https://deploy-preview-338--cobalt-docs.netlify.app/platform-deep-dive/collaboration/organization/organization-settings/saml-sso/okta/) | Guide for the Cobalt SAML app |

## Preview This Change

To see how this change looks in production, scroll down to **Deploy Preview**. Select the link that looks like `https://deploy-preview-<num>--cobalt-docs.netlify.app/`

## Variables

Help us support a “Write once, publish everywhere” single source of truth. If you see a line that looks like:

`{{% asset-categories %}}`

You’ve found a [shortcode](https://gohugo.io/content-management/shortcodes/) that we include in multiple documents.

You’ll find the content of the shortcode in the following directory:

https://github.com/cobalthq/cobalt-product-public-docs/tree/main/layouts/shortcodes

That shortcode has the same base name as what you see in the PR, such as `asset-categories.html`.

## Checklist for PR Author

[ ] Did you check for broken links and alt text?

Be sure to check for broken links and Alt Text issues. We have a partially automated process,
as described in this section of our repository README:
[Test Links and Alt Attributes](https://github.com/cobalthq/cobalt-product-public-docs/blob/main/README.md#test-links-and-alt-attributes).

### CODE REVIEW

File: content/en/Platform Deep Dive/Collaboration/Organization/Organization Settings/SAML SSO/okta.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    - Click **Save**.
+1. Done!
+
+## Notes
+
+The following SAML attributes are supported:
+
+| Name      | Value          |
+| --------- | -------------- |
+| email     | user.userName     |
```
Comment (molfinn) [inline content/en/Platform Deep Dive/Collaboration/Organization/Organization Settings/SAML SSO/okta.md:54]: I think the value here needs to be email not user.userName

File: content/en/Platform Deep Dive/Collaboration/Organization/Organization Settings/SAML SSO/okta.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+1. Sign in to Cobalt, and go to **Settings** > **Security**. You should have an [Organization Owner](/getting-started/glossary/#organization-owner) role.
+1. Under **Configure SAML**, click **Configure**. An overlay for configuring SAML opens.<br><br>
+    ![SAML SSO configuration overlay in the Cobalt app](/deepdive/configure-SAML-overlay.png "SAML SSO configuration overlay in the Cobalt app")
+1. Enter the following values from Okta. In the Okta Admin Dashboard, select the **Sign On** tab for the Cobalt SAML app, then click **Edit**.
+    - **IdP SSO URL**: Enter **Sign on URL**.
+    - **IdP Certificate**: Enter **Signing Certificate**. Download the certificate, and copy the file contents including `-----BEGIN CERTIFICATE-----` and `-----END CERTIFICATE-----`.
+    - Click **Save Configuration**.<br><br>
+    ![Set the Application username format in Okta](/deepdive/Okta-SAML-configurations.png "Set the Application username format in Okta")
+1. In Okta, select the **Sign On** tab for the Cobalt SAML app, then click **Edit**.
+    - **Slug**: Enter your organization’s slug from Cobalt. To copy the slug, in Cobalt, go to **Settings** > **General**.
```
Comment (molfinn) [inline content/en/Platform Deep Dive/Collaboration/Organization/Organization Settings/SAML SSO/okta.md:42]: could also say it' s after the = in the overlay. Example https://login.app.cobalt.io/login/callback?connection=saml-test-org 

the 'saml-test-org' is the slug

File: content/en/Platform Deep Dive/Collaboration/Organization/Organization Settings/SAML SSO/okta.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+- [Supported Features](#supported-features)
+- [Configuration Steps](#configuration-steps)
+- [Notes](#notes)
+
+## Supported Features
+
+The Okta/Cobalt SAML integration supports the following features:
+
+- IdP-initiated SSO
```
[the remark this answers is not in the document]
Comment (ana-dashuk-cobalt) [reply content/en/Platform Deep Dive/Collaboration/Organization/Organization Settings/SAML SSO/okta.md:27]: It's the format that Okta requires.

File: content/en/Platform Deep Dive/Collaboration/Organization/Organization Settings/SAML SSO/okta.md (hunk 1 above)
Comment (ana-dashuk-cobalt) [reply content/en/Platform Deep Dive/Collaboration/Organization/Organization Settings/SAML SSO/okta.md:54]: I asked Okta about this parameter, it must be `user.userName`.

File: content/en/Platform Deep Dive/Collaboration/Organization/Organization Settings/SAML SSO/okta.md (hunk 2 above)
Comment (ana-dashuk-cobalt) [reply content/en/Platform Deep Dive/Collaboration/Organization/Organization Settings/SAML SSO/okta.md]: Updated and added an image.

Screenshot 2023-04-18 at 11 37 00

**Review by molfinn — APPROVED** [review]

**Pull request merged**